src/book/entities/Account.ts

```diff
@@ -243,10 +248,27 @@ function CheckPlaceholder(validationOptions?: v.ValidationOptions) {
             }
           }
 
+          if (account.childrenIds?.length > 0 && !account.placeholder) {
+            return false;
+          }
+
+          if (account.parent?.type === 'INVESTMENT' && account.placeholder) {
+            return false;
+          }
```

Comment:
We don't allow multiple levels of nesting placeholder INVESTMENT accounts. The very top level should be an 'ASSET'